toolkit/templates/forms/_checkbox-tree-selection-buttons.html

```diff
@@ -0,0 +1,71 @@
+<fieldset class="checkbox-tree__selection-buttons">
+  <legend>
+    <h2>{{ name }}</h2>
+  </legend>
+  {% for supercategory in options %}
+    {% set id_prefix = 'input-{}-{}'.format(name, supercategory.label|replace(" ", "")) %}
+    {% if supercategory.options %}
+      <details
+          {% for _o in supercategory.options %}
```

Comment:
It _is_ a bit odd but it's meant to be pretty self-contained and not clash with variable declarations elsewhere.